h2o-persist-gcs/src/main/java/water/persist/PersistGcs.java

```diff
@@ -29,9 +29,25 @@
 /**
  * Persistence backend for GCS
  */
+@SuppressWarnings("unused")
 public final class PersistGcs extends Persist {
 
-  private final Storage storage = StorageOptions.getDefaultInstance().getService();
+  private Storage storage;
```

Comment:
it would be a little safer if this class didn't have a direct access to the field (developer can accidentally use the field directly), I would suggest to move it to a provider class

or use a proxy pattern (guava makes it easy)